### PR TITLE
Fix typo in generic-interfaces.md

### PR DESCRIPTION
**Pull request by focuse**
fixed a simple typo, that is **"contravariant"** is a single word.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/programming-guide/generics/generic-interfaces.md
Comment (rpetrusha) [inline docs/csharp/programming-guide/generics/generic-interfaces.md]: There's a typo in "contrvariant".

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @focuse, for contributing to the dotnet/docs repo and removing the unnecessary hyphen. However, you've also introduced a new typo. Once you fix it, we'll merge your PR.

File: docs/csharp/programming-guide/generics/generic-interfaces.md
Comment (focuse) [reply docs/csharp/programming-guide/generics/generic-interfaces.md]: fixed `contrvariant` -> `contravariant`.

**Review by BillWagner — APPROVED** [review]
Thanks for making this fix @focuse 

@rpetrusha I've reviewed the commit after your comments, and this LGTM. I'll let you approve and :shipit:

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**
